<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>1045</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>173</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_unar">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>173</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>88</y>
      <width>91</width>
      <height>93</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>173</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>256</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>256</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>256</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>256</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>339</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>422</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>422</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>422</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>339</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>422</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>339</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>339</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>505</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>505</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_ravno">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>505</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>256</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>505</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>339</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>173</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>422</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>173</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>256</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>339</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>422</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>505</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>2</y>
      <width>91</width>
      <height>94</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>2</y>
      <width>91</width>
      <height>94</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comma">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>88</y>
      <width>91</width>
      <height>93</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>173</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>2</y>
      <width>21</width>
      <height>594</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_X">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>505</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_numbers</string>
    </attribute>
   </widget>
   <widget class="QTextEdit" name="result_show">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>12</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="pre_result_show">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>85</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>640</y>
      <width>558</width>
      <height>389</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="check_ope_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_enter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>155</y>
      <width>368</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1030</y>
      <width>573</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_credit_show">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>-9</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Credit Calc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deposit_show">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>-9</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposit Calc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>588</y>
      <width>580</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчитать график</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>612</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-50</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>612</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>613</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>614</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>max x</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_numbers"/>
 </buttongroups>
</ui>
